### ISSUE TITLE
.progress-bar-animated ignores $enable-prefers-reduced-motion-media-query.

### ISSUE THREAD

**Issue by hardaqa** [twbs/bootstrap#28528]
.progress-bar-animated ignores $enable-prefers-reduced-motion-media-query

Missing condition
`@if $enable-prefers-reduced-motion-media-query {`
before line 39 in _progress.scss

**Comment by patrickhlauke** [comment]
reduced motion is generally concerned with animations that can cause issues for users with vestibular disorders (i.e. parallax scrolling, slow animated opening/closing things). it's not necessarily a "no animations at all" setting. at this stage, we've left the animated progress bar on purpose, unless there's evidence this can cause issues to users?

**Comment by hardaqa** [comment]
You already did implement reduced-motion media query #27986. I want to enable animations in my design regardless of browser/system settings. Issue is a variable `$enable-prefers-reduced-motion-media-query` having no effect wether it's true or false and leaves the animation disabled on my system.

**Comment by patrickhlauke** [comment]
ah, now i see what you mean - i forgot that we flipped on that one in the end. yes, since we do have a variable for reduced motion, it makes sense to add that there.
